### DISCUSSION TITLE
Firefox Developer Edition always opens in Troubleshooting Mode - what am I doing wrong?

### DISCUSSION THREAD

**Discussion by christiancable** [ttscoff/bunch-docs#108] [Q&A]
I'm not sure what I am doing wrong at all here but each time I try to open Firefox Developer Edition in a bunch it opens in Troubleshooting Mode. 

Firefox Developer Edition is set as my default browser.

Normally it opens as normal. (from icon, shell, applescript etc) 
Here's the commands I have tried in my bunch so far.  

Any ideas on what else I could try or what I am doing wrong?

```
// Using the ID
org.mozilla.firefoxdeveloperedition

// just trying firefox
Firefox

// trying the full name
Firefox Developer Edition

// applescript
* Tell application "/Applications/Firefox Developer Edition.app" to activate

// shell script
$ "/Applications/Firefox Developer Edition.app/Contents/MacOS/firefox"

// opening a url via shell script
$open https://trello.com/<insert actual url here>

// opening a url
https://trello.com/<insert actual url here>

// opening a url via the firefox command - this actually gives me a bunch error dialog telling me it cannot open it
firefox:https://trello.com/<insert actual url here>
```

**Comment by ttscoff** [comment]
I just went to test, and not only did FDE open in troubleshooting mode, 
it was in Spanish. I'm not sure why this would be happening. Like, at 
all. Will have to look into it further. Regular old Firefox doesn't seem 
to have this issue.

-Brett
